Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -301,8 +301,21 @@ extension ErrorMessageGenerator {
     }
   }
   
-  func duplicateExclusiveValues(previous: InputOrigin, duplicate: InputOrigin) -> String? {
-    return "Value at position \(duplicate) has already been set from value at position \(previous)."
+  func duplicateExclusiveValues(previous: InputOrigin, duplicate: InputOrigin, arguments: [String]) -> String? {
+    func elementString(_ origin: InputOrigin, _ arguments: [String]) -> String? {
+      guard case .argumentIndex(let split) = origin.elements.first else { return nil }
+      var argument = "\"\(arguments[split.inputIndex.rawValue])\""
+      if case let .sub(offsetIndex) = split.subIndex {
+        let stringIndex = argument.index(argument.startIndex, offsetBy: offsetIndex+2)
+        argument = "\"\(argument[stringIndex])\" in \(argument)"
+      }
+      return "flag \(argument)"
+    }
+
+    let dupeString = elementString(duplicate, arguments) ?? "position \(duplicate)"
```

Comment:
Can you add to-do comment here to about updating this when we support environment values? It's worth noting that the RHS of these coalescing operators won't ever get hit.

Reply:
Done